### ISSUE TITLE
CAUTION: current branches - please read!

### ISSUE THREAD

**Issue by nesnoj** [openego/data_processing#86]
I noticed that there have been recent commits to [`dev`](https://github.com/openego/data_processing/commits/dev) as well as to [`release/v0.2.6`](https://github.com/openego/data_processing/commits/release/v0.2.6).
To my mind the actual branch to work on is `release/v0.2.6`, right?

I tried to make a pull request but `dev` cannot be automatically merged into `release/v0.2.6`.
As one can see in the [branch network](https://github.com/openego/data_processing/network), there has been 15 commits since we created `release/v0.2.6` from `dev` on 30.03.

**Measures:**
If we continue using `release/v0.2.6` for new commits:
All recent committers in `dev` (@Ludee @lukasol @IlkaCu @ulfmueller) are to checkout `release/v0.2.6` and cherry-pick their changes (if not done yet). There has been changes in branches [`refactor/upgrade-to-pypsa-0.8`](https://github.com/openego/data_processing/tree/refactor/upgrade-to-pypsa-0.8) @s3pp and others which were created from `dev` - this might be a problem since they are not merged back to `dev`.

Opinions? Objections?

**Comment by IlkaCu** [comment]
We pushed all further development relevant for release v0.3 (and irrelevant for v0.2) to the dev branch. 
All bugfixes relevant for release v0.2 need to be pushed to the release branch.
But we should be aware that a merge of both branches will be much more difficult the longer and more intense we work on both branches in parallel.

**Comment by nesnoj** [comment]
Great!
So we are to define a new release date, I propose to do this in the [upcoming WebCo](http://oep.iks.cs.ovgu.de/redmine/projects/open_ego/wiki/Webconf_02052017).

**Comment by IlkaCu** [comment]
v0.2 is released and our branching model got specified during the last project meeting.
